src/com/hillel/Main.java

```diff
@@ -5,5 +5,6 @@ public static void main(String[] args) {
         System.out.println("Hello Java");
         System.out.println("Hello EugeniyaDzhu");
         System.out.println("Hello Java Pro");
+        System.out.println("I hate git...");
```

Comment:
Don't be so categorical :) Please remove your changes from "com.hillel" and create you own package